platform/security/sts/security-sts-ldaplogin/src/main/java/ddf/ldap/ldaplogin/LdapService.java

```diff
@@ -54,10 +53,8 @@ public LdapService(final BundleContext context) {
      * @param newModule that will replace a module or be added to the list of modules.
      */
     public synchronized void update(Module newModule) {
-        modules = modules.stream()
-                .filter(m -> !m.getName()
-                        .equals(newModule.getName()))
-                .collect(Collectors.toList());
+        modules.removeIf(m -> m.getName()
```

Comment:
Looking at this again, it seems like a map would be more appropriate than a list. But this also seems like it's outside the scope of this ticket.